tracer/src/Datadog.Trace/Agent/DiscoveryService/IDiscoveryService.cs

```diff
@@ -12,7 +12,7 @@ namespace Datadog.Trace.Agent.DiscoveryService
     /// </summary>
     internal interface IDiscoveryService
     {
-        string ProbeConfigurationEndpoint { get; }
+        string ConfigurationEndpoint { get; }
 
         string DebuggerEndpoint { get; }
```

Comment:
Propose XML comment:
Returns the endpoint in datadog-agent which is used to upload debugger snapshots, or null if the endpoint is not available.